### DISCUSSION TITLE
Configure the IEEE address of the coordinator?

### DISCUSSION THREAD

**Discussion by teeling** [Koenkk/zigbee2mqtt#9963] [General]
Hi,
I'm planning to update my old CC2531 to a ZZH. I'm reading on the z2m FAQ (https://www.zigbee2mqtt.io/guide/faq/#how-do-i-migrate-from-a-cc2530-cc2531-to-a-more-powerful-coordinator-e-g-zzh) that I should copy the IEEE address from my old coordinator to the new one, but the instructions aren't very clear.
It says: "Using the flashing tool, write the ieee address of your old coordinator to the new coordinator".
Which flashing tool is this, and how would I enter the ieee address?
I was planning to use the cc2538-bsl.py method of flashing firmware, but I can't find any indication in its documentation of where I can supply the ieee address. Should I use any other flashing method?

Thanks!

**Comment by Koenkk** [comment]
This can be done with the `-i` flag: https://github.com/JelmerT/cc2538-bsl/blob/0daa5d0980eae6a9eb58693c05503fdf6538703b/cc2538-bsl.py#L1041

**Reply by teeling** [reply]
Perfect, thanks!

**Reply by captaindaveman** [reply]
I actually have the same question but I'm using Flash Programmer 2 to flash the device (SONOFF Zigbee 3.0 USB Dongle Plus)

**Reply by Koenkk** [reply]
I think you can do this via the "MAC Address" tab:

![image](https://user-images.githubusercontent.com/2892853/144745969-319b3827-dce3-4fcf-96c5-93e57aac1827.png)

**Reply by sbcrumb** [reply]
If any is wonder the above did work using the SmartRF Flash. I just did it worked no issues.
